Presentation.Web/app/models/ui-customization/ui-customization.ts

```diff
@@ -168,6 +179,10 @@
             this.children.forEach(child => child.accept(visitor));
         }
 
+        acceptChildren(visitor: IUICustomizationTreeVisitor, node: IUINode): void {
```

Comment:
make this one private since it is a private method and not supposed to be used from the outside